src/services/saneKeyboardEvents.util.js

```diff
@@ -234,27 +290,40 @@ var saneKeyboardEvents = (function() {
       //
       // And by nifty, we mean dumb (but useful sometimes).
       if (document.activeElement !== textarea[0]) {
-        textarea.focus();
+        textarea[0].focus();
       }
 
       checkTextareaFor(pastedText);
     }
     function pastedText() {
       var text = textarea.val();
       textarea.val('');
-      if (text) handlers.paste(text);
+      if (text) controller.paste(text);
     }
 
     // -*- attach event handlers -*- //
-    target.bind({
-      keydown: onKeydown,
-      keypress: onKeypress,
-      keyup: onKeyup,
-      focusout: onBlur,
-      cut: function() { checkTextareaOnce(function() { handlers.cut(); }); },
-      copy: function() { checkTextareaOnce(function() { handlers.copy(); }); },
-      paste: onPaste
-    });
+
+    if (controller.options && controller.options.disableCopyPaste) {
```

Comment:
To shorten this a bit, we might want to just create named functions for onCopy, onCut, onPaste, and only call target.bind() once. The functions could either check the config value disableCopyPaste to change their behavior, or we could conditionally assigning different functions to those names once based on the config value.

It's not a big thing, I can file a follow up for it, but just wanted to mention it, this duplicated code also gets a bit longer now that we have prettier and can't have single line function declarations.

Reply:
I think this could also be shortened to `if (controller.options?.disableCopyPaste)`